target/linux/ipq40xx/image/Makefile

```diff
@@ -164,6 +164,18 @@ define Device/avm_fritzrepeater-3000
 endef
 TARGET_DEVICES += avm_fritzrepeater-3000
 
+define Device/cilab_meshpoint-one
+	$(call Device/FitImage)
```

Comment:
With `$(call Device/8dev_jalapeno)` you can follow DRY principle and reuse 4 common variables, as a bonus you're making it explicit, that this device is based on Jalapeno.

Reply:
I'm not a big fan of inheriting devices from other Devices. This drove me mad in ar71xx already, and I'm glad this stopped in ath79.
However, with 4 common variables it might qualify for a common definition that can be inherited by both devices (like it's done for Device/netgear_ex61x0v2)...